MathLib/LinAlg/Eigen/EigenLinearSolver.cpp

```diff
@@ -265,6 +293,18 @@ void EigenLinearSolver::setOption(BaseLib::ConfigTree const& option)
             "scaling is not available.");
 #endif
     }
+    if (auto restart =
+            //! \ogs_file_param{prj__linear_solvers__linear_solver__eigen__restart}
```

Comment:
tag documentation is required. Preferably refer to the documentation of the `EigenOption::restart` member.